server/lib/activities.js

```diff
@@ -175,7 +175,14 @@ const doFormatMessage = (activity, format) => {
       return `New ${update} published on ${collective}`;
 
     case activities.COLLECTIVE_MEMBER_CREATED:
-      return `New member ${member} joined ${collective}`;
+      if (amount) {
+        return `${member} just joined ${collective} and contributed  with ${formatCurrency(
+          currency,
+          amount,
+        )}! Well done and thank you very much!`;
```

Comment:
:ice_cream: Not sure we need to bring back that part (my personal opinion)

```suggestion
        )}`;
```

Reply:
Makes sense. I've felt the same. I've removed that part. 👍🏽